README.md

```diff
@@ -17,58 +17,82 @@ option.
 
 ### Compute
 
-Computes coverage from `target/catalog.json` and `target/manifest.json` files
-found in a dbt project. You need to select documentation or test coverage by
-using the respective CLI argument.
+Compute coverage from `target/catalog.json` and `target/manifest.json` files
+found in a dbt project, e.g.
+[jaffle_shop](https://github.com/dbt-labs/jaffle_shop). You need to select
+documentation or test coverage by using the respective CLI argument.
 
 ```
-$ cd my-dbt-project
+$ cd jaffle_shop
+$ dbt run  # Materialize models
 $ dbt docs generate  # Generate catalog.json and manifest.json
 $ dbt-coverage compute doc  # Compute doc coverage, print it and write it to coverage.json file
-$ dbt-coverage compute test  # Compute test coverage
-
-# Coverage summary
-# =======================================
-# my-table-1               8/20     40.0%
-# my-table-2              20/20    100.0%
-# my-table-3               2/8      25.0%
-# =======================================
-# Total                   30/48     62.5%
+
+# Coverage report
```

Comment:
@sweco is there a reason why you are commenting these out? I don't think it looks like that in the output, right?

Reply:
I guess in the spirit of beefing this up, some https://asciinema.org/ might be worth doing (but is certainly not necessary 🙂 )

Reply:
I'll uncomment these and write `asciinema` down for beefing up. 😄

Reply:
Done here: https://github.com/slidoapp/dbt-coverage/pull/11